.ci/.jenkins_framework.yml

```diff
@@ -36,6 +36,7 @@ FRAMEWORK:
   - mysqlclient-newest
   - aiohttp-newest
   - aiopg-newest
+  - asyncpg-newest
```

Comment:
In addition to this, you'll have to add entries into `.ci/jenkins_exclude.yml` for Python 2.7, 3.5 and 3.6, as well as pypy-2 and pypy-3, analogue to the aiopg entries:

https://github.com/elastic/apm-agent-python/blob/b3b08c282dbd4949129736195df38d2258ee7503/.ci/.jenkins_exclude.yml#L259-L269